### PR TITLE
ISSUE-23 Anastasia Voloshina

### PR DESCRIPTION
**Pull request by a-voloshina**
CSC profile link: https://my.compscicenter.ru/users/6093/

Решила пример 23 правильно
Решила пример 23 неправильно
Изменила пример 29

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -20,13 +20,13 @@
 20 + 63
 64 + 87
 66 - 68 = -2
-41 - 69
+41 - 69 = 28
```
Comment (acidcoma) [inline tasks]: please correct your answer

Comment (a-voloshina) [reply tasks]: Answer is corrected

Comment (acidcoma) [reply tasks]: good job!

**Review by acidcoma — APPROVED** [review]

**Comment by a-voloshina** [comment]
My pull request: https://github.com/cscenter/practical-minimum-2020-students/pull/68/
Colleague pull request: https://github.com/cscenter/practical-minimum-2020-students/pull/88

**Review by dehasi — APPROVED** [review]

**Pull request merged**